Source/Scene/BufferLoader.js

```diff
@@ -0,0 +1,158 @@
+import defaultValue from "../Core/defaultValue.js";
```

Comment:
Meta thing: I'm wondering if this `XxxxLoader` convention should really be `XxxxResourceLoader`

I was looking through the code coverage report and got confused when I saw a file named `DracoLoader`, which is NOT the glTF draco loader :sweat_smile:

Reply:
I pefer `XxxxLoader` just so it's less wordy.

Maybe `DracoLoader` should be renamed to `DracoDecoder`...